src/EventGridExtension/OutputBinding/EventGridAsyncCollector.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
```

Comment:
Please add file headers to these files (see EventGridTriggerAttribute). Unfortunately it looks like file headers are missing from other existing files :(